dist/index.js

```diff
@@ -7189,8 +7189,8 @@ async function main() {
   const rebar3Spec = core.getInput('rebar3-version', { required: false })
 
   if (otpSpec !== 'false') {
-    const otpVersion = await installOTP(otpSpec, osVersion)
```

Comment:
We install Elixir based on the direct OTP input, and not the output of this function (easier to reason about)...

Reply:
Nice one!